apps/backend/src/lambdas/helpers.ts

```diff
@@ -28,25 +30,31 @@ export const validateSbcGarmsFileName = (filename: string): boolean => {
  * @param message
  * @returns
  */
-export const generateLocalSNSMessage = (message: unknown) => {
+/**
+ * Used to generate a local SNS message for running lambdas locally
```

Comment:
I'm wondering if this wasn't working until now, given the defaults you had to add to sns-manager.service.ts ?

Reply:
Previously we were using the "nest-aws-sdk" (open source pkg) to set those defaults in the app module and they were available in the services using the "@InjectAWS" decorator. That pkg was not compatible with v3. There is a similar pkg we could use instead: [here](https://www.npmjs.com/package/aws-sdk-v3-nest) but it didn't seem necessary....

Reply:
oh, also, this doesn't _really_ generate a local SNS message...I think we would have to use local stack for that. This will just call the handler with the same type as the SNS message that is passed in on dev/test/prod